Source/Main.as

```diff
@@ -53,6 +63,10 @@ void RenderInterface()
 	}
 }
 
+void Update(float dt) {
+	g_dt = dt;
+}
+
```

Comment:
Formatting on this function is wrong, it should be:
```
void Update(float dt)
{
}
```
Also, probably put a comment here saying what the delta time is required for.